debezium-connector-ibmi/src/main/java/io/debezium/connector/db2as400/As400ValueConverters.java

```diff
@@ -24,6 +26,10 @@ public class As400ValueConverters extends JdbcValueConverters {
     public As400ValueConverters() {
```

Comment:
should we delete the decault constructor? It doesn't look like it's used anywhere else